extensions/readwise/src/components/ResultsList.tsx

```diff
@@ -10,7 +10,11 @@ import { getListSubtitle } from "../utils";
  * Note that the EmptyView is never displayed if the List’s isLoading.
  */
 export const ListWithEmptyView = () => (
-  <List.EmptyView title="No results" description="Type something to search again." icon={{ source: Icon.Binoculars }} />
+  <List.EmptyView
+    title="No Results"
+    description="Enter something to search again."
```

Comment:
```suggestion
```
Since it's used in multiple contexts, it would be better to just show the title – otherwise it should be a bit more context aware by making the description dynamic 😄